journals/apps/journals/models.py

```diff
@@ -507,6 +507,8 @@ class JournalPage(JournalPageMixin, Page):
     A page inside a journal. These can be nested indefinitely. Restricted to
     users who purchased access to the journal.
     """
+    journal_about_page = models.ForeignKey(JournalAboutPage, null=True, blank=True, related_name='journal_pages', on_delete=models.SET_NULL)
```

Comment:
what is logic behind naming `related_name` to `journal_pages`?

Reply:
related_name lets you set the name of the reverse lookup. With it, you can access all the pages attached to the journal about page via `journalaboutpage.journal_pages`.